### PR TITLE
Add changelog for Kubernetes 1.22

### PR DESCRIPTION
**Pull request by bl0m1**

### CODE REVIEW

**Review by qvicksilver — APPROVED** [review]
LGTM

File: content/en/docs/private-kubernetes/changelog/v2.18.1-elx1.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+* All new cluster will have the cluster domain `cluster.local` by default
+* All encrypted (`*-enc`) storageclasses (`4k-enc`, `8k-enc` and `16k-enc`) are no longer applied to new clusters since they are deprecated for removal in Openstack. Do not worry the 4k, 8k and 16k storage-classes are encrypted. [Read our full announcement here](https://docs.elastx.cloud/docs/openstack-iaas/announcement/#2022-01-17)
+
+## Deprecations
+
+Note that all deprecations will be removed in a future Kubernetes release, this does not mean you need to perform any changes now however we recommend you to start migrating your applications to avoid issues in future releases.
+
+### APIs removed in Kubernetes 1.25
+
+more detail can be found [in Kubernetes official documentation.](https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-25)
```
Comment (Doctor-love) [inline content/en/docs/private-kubernetes/changelog/v2.18.1-elx1.md:33]: Fix casing.

File: content/en/docs/private-kubernetes/changelog/v2.18.1-elx1.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+* Certmanager: 1.6.3
+
+## Major changes
+
+* When our ingress is installed we set it as the default ingress meaning it will be used unless a customer ingress class is used/specified.
+* Clusters are now running containerd instead of docker. This should not affect your workload at all.
+* We reserve 5% RAM on all nodes making it easier to calculate how much is left for your workload.
+* All components deployed by Elastx have tolerations for `NoSchedule` taints by default
+* Certmanager `cert-manager.io/v1alpha2`, `cert-manager.io/v1alpha3`, `cert-manager.io/v1beta1`, `acme.cert-manager.io/v1alpha2`, `acme.cert-manager.io/v1alpha3` and `acme.cert-manager.io/v1beta1` APIs are no longer serverd, all existing resources will be converted automatically to `cert-manager.io/v1` and `acme.cert-manager.io/v1` however you still need to update your local manifests
+* A list of old APIs are now longer served. [A complete list can be found in Kubernetes documentation](https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-22)
```
Comment (Doctor-love) [inline content/en/docs/private-kubernetes/changelog/v2.18.1-elx1.md:20]: "A number of" or perhaps "Several"?

File: content/en/docs/private-kubernetes/changelog/v2.18.1-elx1.md (hunk 1 above)
Comment (bl0m1) [reply content/en/docs/private-kubernetes/changelog/v2.18.1-elx1.md]: Fixed :)

File: content/en/docs/private-kubernetes/changelog/v2.18.1-elx1.md (hunk 2 above)
Comment (bl0m1) [reply content/en/docs/private-kubernetes/changelog/v2.18.1-elx1.md]: Fixed :)

**Pull request merged**